apps/betterangels-backend/clients/tests/test_permissions.py

```diff
@@ -132,13 +132,15 @@ def test_view_client_profiles_permission(self, user_label: str, should_succeed:
         client_count = ClientProfile.objects.count()
         mutation = """
             query ViewClientProfiles {
-                clientProfiles {
-                    id
-                    user {
-                        firstName
-                        lastName
-                        middleName
-                        email
+                clientProfilesPaginated(pagination: {offset: 0, limit: 10}) {
```

Comment:
`pagination` is an optional argument that only has to be passed when needed. We don't need to paginate these results, so it's ok to omit `paginated`
```suggestion
                clientProfilesPaginated {
```